monkey/monkey_island/cc/services/attack/technique_reports/T1156.py

```diff
@@ -20,8 +20,8 @@ class T1156(AttackTechnique):
                                        'ips': [{'$arrayElemAt': ['$data.ip', 0]}]},
                            'result': '$data.result'}},
              {'$unwind': '$result'},
-             {'$match': {'$or': [{'result': {'$regex': '\.bash'}},
-                                 {'result': {'$regex': '\.profile'}}]}}]
+             {'$match': {'$or': [{'result': {'$regex': '\.bash'}},  # noqa: W605
+                                 {'result': {'$regex': '\.profile'}}]}}]  # noqa: W605
```

Comment:
Why are we ignoring the errors here? I think this is actually an invalid escape sequence, we probably want to precede it with an `r` (or fix the regex).
https://www.flake8rules.com/rules/W605.html